### PR TITLE
Waf 19/plp filters 

### PR DESCRIPTION
**Pull request by johnnyung91**
This PR addresses overall filter structure across desktop and mobile

## Description
Added overall layout based on new designs for desktop and mobile features.
- Separated logic on desktop filters into heading names and facets
- Mobile drawer complete, although filtering logic needs to be reworked
- Added more Tailwind classes

### CODE REVIEW

**Review by chicharr — APPROVED** [review]
LGTM

**Comment by chicharr** [comment]
Hey @johnnyung91 ,
please fix the conflicts :)

**Review by yugandhar02 — APPROVED** [review]
code wise LGTM
there are few bugs

1. Layout is broken when styling filter is selected

image

2) "+" icon is not toggled when category menu is open
image

**Comment by johnnyung91** [comment]
@yugandhar02 the layout bug i will address it in another PR since i am unable to recreate it.  lets go ahead and merge this for now?

File: src/components/Facets/format.ts [hunk 1]
```diff
@@ -40,6 +40,7 @@ const formatBinaryLabel = (
   categoryPath?: string
 ) => {
   if (categoryPath && categoryNames) {
+    console.log('working2')
```
Comment (yugandhar02) [inline src/components/Facets/format.ts:43]: remove unnecessary console log ?

**Pull request merged**